packages/vitest/src/runtime/execute.ts

```diff
@@ -313,6 +320,8 @@ export class VitestExecutor extends ViteNodeRunner {
       columnOffset: -codeDefinition.length,
     }
 
+    this.options.moduleExecutionInfo?.set(options.filename, { startOffset: codeDefinition.length })
```

Comment:
Not sure if `filename` is a good choice here. Wouldn't `url` be better? It should include queries if there are any

Reply:
Now as it's using `moduleCache`, the key must match that one. I think the `options.filename` here contains query params too 🤔

Reply:
`moduleCache` doesn't use `filename` tho, it uses normalized module id

Reply:
Ok, I'll add test case that loads Worker with query params later.

This change should break Vue tests if query params are missing. 🤔

Reply:
The used id/filename must match the one that is being passed as `filename` in `vm.runInThisContext`. That's what V8 sets in the runtime results.

> moduleCache doesn't use filename tho, it uses normalized module id

When are these two different? Can't see any differences between these two in the tests.